.github/workflows/release.yml

```diff
@@ -84,20 +86,24 @@ jobs:
               generate_release_notes: true
             })
 
-      - name: Check commit messages for feat or break
+      - name: Check commit messages and descriptions for feat or break
         id: check_commits
         run: |
```

Comment:
The name of the step "Check commit messages and descriptions for feat or break" is too long and could be simplified for better readability.